CHANGELOG.md

```diff
@@ -5,68 +5,70 @@
 
 BUGFIXES:
-* [1179](https://github.com/nginxinc/kubernetes-ingress/pull/1179) Fix TransportServers in debian AppProtect image
-* [1129](https://github.com/nginxinc/kubernetes-ingress/pull/1129) Support real-ip in default server
-* [1110](https://github.com/nginxinc/kubernetes-ingress/pull/1110) Add missing threat campaigns key to AppProtect CRD
-* [1078](https://github.com/nginxinc/kubernetes-ingress/pull/1078) Fix trailing blank lines in vs/vsr snippets
+* [1179](https://github.com/nginxinc/kubernetes-ingress/pull/1179) Fix TransportServers in debian AppProtect image.
+* [1129](https://github.com/nginxinc/kubernetes-ingress/pull/1129) Support real-ip in default server.
+* [1110](https://github.com/nginxinc/kubernetes-ingress/pull/1110) Add missing threat campaigns key to AppProtect CRD.
 
 HELM CHART:
-* The version of the helm chart is now 0.6.2
-* [1105](https://github.com/nginxinc/kubernetes-ingress/pull/1105) Fix GlobalConfiguration support in helm chart
+* The version of the helm chart is now 0.7.0
+* [1105](https://github.com/nginxinc/kubernetes-ingress/pull/1105) Fix GlobalConfiguration support in helm chart.
+* Add new parameters to the Chart: `controller.setAsDefaultIngress`, `controller.enableLatencyMetrics`. Added in [1133](https://github.com/nginxinc/kubernetes-ingress/pull/1133) and [1148](https://github.com/nginxinc/kubernetes-ingress/pull/1148).
 
 CHANGES:
 * [1182](https://github.com/nginxinc/kubernetes-ingress/pull/1182) Update NGINX version to 1.19.3.
 
 UPGRADE:
 * For NGINX, use the 1.9.0 image from our DockerHub: `nginx/nginx-ingress:1.9.0`, `nginx/nginx-ingress:1.9.0-alpine` or `nginx-ingress:1.9.0-ubi`
 * For NGINX Plus, please build your own image using the 1.9.0 source code.
-* For Helm, use version 0.6.2 of the chart.
+* For Helm, use version 0.7.0 of the chart.
 
-NOTES:
-* When using version of k8s >= 1.18 the `-use-ingress-class-only` cli arg is now ignored and the Ingress Controller will only handle resources with the appropriate class set.
-* A unique ingressClass is required per helm release. When making additional releases ensure `controller.ingressClass` is not set to the name of an already existing ingressClass.
+For Kubernetes >= 1.18, when upgrading using the [manifests](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/), make sure to update the [ClusterRole](deployments/rbac/rbac.yaml) and create the [IngressClass resource](deployments/common/ingress-class.yaml), which is required for Kubernetes >= 1.18. Otherwise, the Ingress Controller will fail to start. If you run multiple NGINX Ingress Controllers in the cluster, each Ingress Controller has to have its own IngressClass resource.
```

Comment:
```suggestion
For Kubernetes >= 1.18, when upgrading using the [manifests](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/), make sure to update the [ClusterRole](deployments/rbac/rbac.yaml) and create the [IngressClass resource](deployments/common/ingress-class.yaml), which is required for Kubernetes >= 1.18. Otherwise, the Ingress Controller will fail to start. If you run multiple NGINX Ingress Controllers in the cluster, each Ingress Controller has to have its own IngressClass resource. As the `-use-ingress-class-only` argument is now ignored (see NOTES), make sure your Ingress resources have the `ingressClassName` field or the `kubernetes.io/ingress.class` annotation set to the name of the IngressClass resource. Otherwise, the Ingress Controller will ignore them.